### ISSUE TITLE
Zero zoom is possible, crashes script

### ISSUE THREAD

**Issue by perey** [HermannBjorgvin/Gravity-Experiments#16]
## Steps to reproduce
1. Load page.
2. Reduce zoom until it reads zero.
3. Crash.

Firefox specifically reports this error:

> A script on this page may be busy, or it may have stopped responding. You can stop the script now, open the script in the debugger, or let the script continue.
> 
> Script: http://hermann.is/gravity/js/gravity/modules/render.js:273
## Expected behaviour

Form field does not allow setting zero zoom. Script enforces a minimum value on zoom, or else raises an error if given zero zoom.

**Comment by MuhsinFatih** [comment]
Thanks for reporting.
@HermannBjorgvin after my commit with zoom it was fine. Problem should be in the latest commits. Expected behavior does not have to be disabling zero zoom.

**Comment by HermannBjorgvin** [comment]
Yes, currently working on a new GUI which combines both the render and gui module, should be much smoother and bug free when implemented. Having the seperate gui and render modules was just something I decided on after not sleeping for 24h and has since proven to be a bad idea and prone to frequent bugs
